### PR TITLE
Release v0.1.0

### PR DESCRIPTION
**Pull request by forrestfwilliams**

### CODE REVIEW

**Comment by forrestfwilliams** [comment]
First version of both metadata and short wavelength tile generation. Both are look well. Screenshots are for descending metadata tiles and ascending short wavelength tiles.
descending_meta
ascending_sw

**Comment by forrestfwilliams** [comment]
> First version of both metadata and short wavelength tile generation. Both are look well. Screenshots are for descending metadata tiles and ascending short wavelength tiles. descending_meta ascending_sw

Gaps in the short wavelength mosaic are due to a lack of data in the California test dataset.

**Comment by forrestfwilliams** [comment]
Gaps in the short wavelength mosaic are due to a lack of data in the California test dataset.

File: src/opera_disp_tms/generate_sw_disp_tile.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    metadata.update(secondary_dates)
+    ds.SetMetadata(metadata)
+    ds.FlushCache()
+    ds = None
+    print('Done!')
+    return product_path
+
+
+def main():
+    """CLI entrpypoint
```
Comment (jacquelynsmale) [inline src/opera_disp_tms/generate_sw_disp_tile.py:230]:
```suggestion
    """CLI entry point
```

File: src/opera_disp_tms/generate_sw_disp_tile.py [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+    Args:
+        metadata_path (Path): Path to the metadata GeoTiff file
+
+    Returns:
+        Dictionary of frame metadata indexed by frame id
+    """
+    info_dict = gdal.Info(str(metadata_path), format='json')
+    frame_metadata = info_dict['metadata']['']
+    frame_ids = [int(x) for x in frame_metadata['OPERA_FRAMES'].split(', ')]
```
Comment (jacquelynsmale) [inline src/opera_disp_tms/generate_sw_disp_tile.py:64]: Do we want to (now or later) handle items with no `frame_metadata['OPERA_FRAMES']`? Even just outputting a list of which frames/tiles aren't generated could be helpful!

Comment (forrestfwilliams) [reply src/opera_disp_tms/generate_sw_disp_tile.py:64]: I think we should not create the metadata tile in this case, so generate_sw_disp_tile.py shouldn't have to handle it.

**Review by jacquelynsmale — APPROVED** [review]

**Pull request merged**